docs/dev/build-instructions/build_macos.md

```diff
@@ -32,7 +32,7 @@ It is recommended to perform a so-called "out-of-source-build". In order to do s
 following commands:
 1. `mkdir build` (Creates a build directory)
 2. `cd build` (Switches into the build directory)
-3. `cmake ..` (Actually runs cmake)
+3. `cmake .. -Dwarnings-as-errors=OFF` (Actually runs cmake)
```

Comment:
This should not be part of the commit for two reasons:
1. We don't generally want to advise people to disable warnings
2. Even if we wanted to, this would have to be a separate commit of type `DOC`

Reply:
If you want to, you can create an entry in the build-faq that tells people that they can disable warnings-as-errors if they encounter a build failure related to that (in a separate commit)

Reply:
This does not disable warnings, it just changes their severity from breaking to display.

Reply:
Yes I know - but let's be honest: It will cause people to ignore warnings when doing their development. That's not what we want.